newsletter-april-2021.rst

```diff
@@ -0,0 +1,116 @@
+
+This is the first of our monthly newsletters, in which we would like to
+openly share with you the most relevant updates of Read the Docs,
+offer a summary of what new features we shipped to our users
+during the previous month,
+and share what things we will be focusing on in the near future.
+
+Company highlights
+------------------
+
+-  We have a new colleague! `Juan Luis`_
+   will be working with us as Developer Advocate, with a focus on fulfilling
+   the goals of `the CZI grant we were
+   awarded <https://blog.readthedocs.com/czi-grant-announcement/>`_,
+   improve our public facing documentation,
+   and spread the word about our service.
+-  The average daily pageviews went up by 9.4 % from the previous month.
+-  We have `a new status page <http://status.readthedocs.com/>`_ so you
+   can see the availability history and current status of our infrastructure.
+
+New features
+------------
+
+-  There is a `new support form <https://docs.readthedocs.io/en/latest/support.html>`_
+   in case you need to send us a request related to your project or
+   account, like asking for more resources, change the project slug, and
+   so forth.
+-  You can now build your project using Python 3.9 by setting your build
+   image to ``testing``. Please `open an
+   issue <https://github.com/readthedocs/readthedocs.org/issues/new>`_
+   if you find any problems with it.
+-  Now `all newly created MkDocs projects use the latest version of mkdocs by
+   default <https://github.com/readthedocs/readthedocs.org/pull/7869>`_.
+   Existing projects can specify a specific version, but won't automatically
+   be upgraded.
+-  We expanded a few areas of our user documentation: we added `a brand
+   new guide on reproducible
+   builds <https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html>`_,
+   considerably improved our `guide on using
+   conda <https://docs.readthedocs.io/en/stable/guides/conda.html>`_,
+   and `started recommending MyST for writing Markdown with
+   Sphinx <https://docs.readthedocs.io/en/stable/intro/getting-started-with-sphinx.html#using-markdown-with-sphinx>`_.
+-  We added the possibility to declare `publicly visible environment
+   variables <https://github.com/readthedocs/readthedocs.org/pull/7891>`_.
+
+You can always see the latest changes to our platforms in our `Read the Docs
+Changelog <https://docs.readthedocs.io/page/changelog.html>`_ and `Ethical Ad Server
+Changelog <https://ethical-ad-server.readthedocs.io/page/developer/changelog.html>`_.
+
+Current focus & known issues
+----------------------------
+
+-  We have had some availability issues with our commercial hosting for the past
+   few weeks. This was caused by massive I/O load spikes on our web servers,
+   causing increased CPU contention and slower web requests. We tracked the issue
+   down to a background process that was doing a lot of reads from our cloud
+   storage. We have changed our infrastructure so background tasks and web
```

Comment:
I think the read were from disk not storage, IIRC

Reply:
Yea, I know, but the issue appeared to be reading from storage (since the data isn't on disk). I noted it was EBS reads, which are the networked disks. That's why it's a mystery.